modules/common/util/factory.h

```diff
@@ -65,7 +65,7 @@ class Factory {
    * @param id Identifier of the class being registered
    * @param creator Function returning a pointer to an instance of
    * the registered class
-   * @return True iff the key id is still available
+   * @return True if the key id is still available
```

Comment:
`iff` sometimes means `if and only if`. I'm not sure if it applies here.

Reply:
Let us confirm